strings/tips.txt

```diff
@@ -55,7 +55,8 @@ As a Scientist, you can disable anomalies by scanning them with an analyzer, the
 As a Scientist, researchable stock parts can seriously improve the efficiency and speed of machines around the station. In some cases, it can even unlock new functions.
 As a Scientist, you can generate research points by letting the tachyon-doppler array record increasingly large explosions.
 As a Scientist, getting drunk just enough will speed up research. Skol!
-As a Scientist, you can get points by placing slime cores into the deconstructive analyzer! This even works with crossbred slime cores. 
+As a Scientist, you can get points by placing slime cores into the deconstructive analyzer! This even works with crossbred slime cores.
+As a Scientist, work with botanists to get different types of seeds, as each type of seed can be used in the deconstructive analyzer for a small amount of Rnd type points!
```

Comment:
Merge the two tips together: `As a Scientist, research points can be generated by deconstructing objects such as slime cores, plant seeds and anomaly cores with the destructive analyzer!`

Reply:
Those are two wildly different tips

Reply:
No, both can be analyzed with the destructive analyzer and provide points, there is nothing wildly different, Trilby.

Reply:
They are different tips, end of story